setup.py

```diff
@@ -48,6 +48,7 @@
           'plone.app.robotframework',
           'plone.app.testing [robot] >= 4.2.2',
           'requests',
+          'Products.ATContentTypes',
```

Comment:
This would definitely need to be an optional dependency. Imagine a DX-only Plone 5+ site, we don't want to pull in Archetypes with plone.restapi.